docs/language/context.md

```diff
@@ -71,13 +71,13 @@ content text {
 }
 ```
 
-will render `Hello, World!`.
+content block will render `Hello, World!`.
 
 ### Inheritance
 
-Variables defined in a parent block (such as `document` or `section`) are available for use in nested
-blocks. Nested blocks can redefine a variable in their context, overwriting the parent variable's
-value.
+Variables defined in a parent block (such as `document` or `section`) are available for use in
+nested blocks. Note that nested blocks can redefine a variable in their context, overwriting the
+parent variable's value.
```

Comment:
I'd reserve "overwriting" for the ref-vars situation, and use shadowing in the nesting situation, it seems more intuitive.
```suggestion
nested blocks. Note that nested blocks can redefine a variable in their context, shadowing the
parent variable's value inside their scope.
```